src/Sylius/Bundle/SettingsBundle/DependencyInjection/Compiler/RegisterResolversPass.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\SettingsBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Steffen Brem <steffenbrem@gmail.com>
+ */
+class RegisterResolversPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('sylius.settings.resolver_registry')) {
```

Comment:
``sylius.registry.settings_resolver``, ``sylius.registry.settings_schema``, we generally avoid prefixing services with bundle name, to keep things consistent with:

``sylius.repository.tax_rate``, ``sylius.manager.shipping_category``, ``sylius.controller.product``, etc. As developer using Sylius you don't want to think from which bundle the service comes.